services-examples/src/main/java/io/scalecube/services/examples/exceptions/ExceptionMapperExample.java

```diff
@@ -15,6 +21,9 @@
   public static void main(String[] args) throws InterruptedException {
     Microservices ms1 =
         Microservices.builder()
+            .discovery(
+                (serviceRegistry, serviceEndpoint) ->
+                    new ScalecubeServiceDiscovery(serviceRegistry, serviceEndpoint).start())
```

Comment:
IMHO i think we can take the time and not pass the service registry to the service discovery
we should pass services we want discovery to publish when starting 
the bridge between service registry and discovery should be in microservices.

Reply:
This is refactoring stage, I keep functionalty without changes , without introducing new concepts.